File: js/lv0/OX퀴즈.js
//[level 0] OX퀴즈 120907
//https://school.programmers.co.kr/learn/courses/30/lessons/120907

//결과 
//정확성: 100.0
//합계: 100.0 / 100.0

function solution(quiz) {
    var result = [];
    
    for (let q of quiz) {
        chk = q.split(' ');
        var a = (chk[1] === '+')? Number(chk[0]) + Number(chk[2]) : Number(chk[0]) - Number(chk[2]);
        
        result.push((a === Number(chk[4]))? 'O' : 'X');    
    }
    
    return result ;
}

File: py/Level0/OX퀴즈.py
#
# [level 0] OX퀴즈
# https://school.programmers.co.kr/learn/courses/30/lessons/120907

# 결과
# 정확성: 100.0
# 합계: 100.0 / 100.0

def solution(quiz):
    result = []
    for q in quiz:
        chk = list(q.split(' '))

        if chk[1] == '+':
            x = int(chk[0]) + int(chk[2])
        else:
            x = int(chk[0]) - int(chk[2])

        result.append("O" if x == int(chk[4]) else "X")

    return result